requirements.txt

```
@@ -31,7 +31,8 @@
 # 3. Use this command to remove per-version files
 #    `rm requirements-?.?.txt`
 #
-cryptography
+cryptography; python_version >= '3'
+cryptography < 3.4; python_version < '3'
```

```suggestion
cryptography >=3.3.2; python_version >= '3'
cryptography >=3.3.2, < 3.4; python_version < '3'
```
... for CVE-2020-36242?